docs/source/config_file.rst

```diff
@@ -211,7 +211,7 @@ section of the command line docs.
     match the name of the *imported* module, not the module containing the
     import statement.
 
-``follow_imports_for_stubs`` (bool, default False)
+``follow_imports_for_stubs`` (bool, default True)
```

Comment:
This is not true. Mypy by default ignores `--follow-import` for stub files (so that they are always `'normal'`).

Reply:
I'm amazed but from my persperctive THIS is not true, and by default `mypy` uses `--follow-import` setting for stubs.

Reply:
Oh, now I see what's going on.
`follow_imports_for_stubs=True` was specified in my config file. :(
I'll fix it, surely.

Reply:
Fixed.